openpype/hosts/unreal/plugins/load/load_layout_existing.py

```diff
@@ -221,41 +222,66 @@ def _load_asset(self, representation, version, instance_name, family):
 
         return assets
 
-    def _process(self, lib_path):
-        data = get_current_project_settings()
-        delete_unmatched = data["unreal"]["delete_unmatched_assets"]
+    def _get_valid_repre_docs(self, project_name, version_ids):
+        valid_formats = ['fbx', 'abc']
 
+        repre_docs = list(get_representations(
+            project_name,
+            representation_names=valid_formats,
+            version_ids=version_ids
+        ))
+        repre_doc_by_version_id = {}
+        for repre_doc in repre_docs:
+            version_id = str(repre_doc["parent"])
+            repre_doc_by_version_id[version_id] = repre_doc
+        return repre_doc_by_version_id
+
+    def _process(self, lib_path, project_name):
         ar = unreal.AssetRegistryHelpers.get_asset_registry()
 
         actors = EditorLevelLibrary.get_all_level_actors()
 
         with open(lib_path, "r") as fp:
             data = json.load(fp)
 
-        layout_data = []
-
+        elements = []
+        repre_ids = set()
         # Get all the representations in the JSON from the database.
         for element in data:
-            if element.get('representation'):
-                layout_data.append((
-                    pipeline.legacy_io.find_one({
-                        "_id": ObjectId(element.get('representation'))
-                    }),
-                    element
-                ))
-
-        containers = []
-        actors_matched = []
+            repre_id = element.get('representation')
+            if repre_id:
+                repre_ids.append(repre_id)
```

Comment:
```suggestion
                repre_ids.add(repre_id)
```

Apart from this, it seems to be working fine!